src/perception/point.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <cstdint>
+
+namespace mrover {
+
+    /**
+     * @brief Defines one element in the point cloud
+     */
+    struct Point {
```

Comment:
I do think its really important to have a comment here explaining that you can't just add stuff to this / really use it in just any context and why.

Reply:
Ok I agree let me make it more clear

Reply:
I dont think you've addressed this yet